geode-core/src/main/java/org/apache/geode/management/internal/beans/CacheServerBridge.java

```diff
@@ -405,7 +405,7 @@ public String toString() {
   }
 
   public Version getClientVersion(ClientConnInfo connInfo) {
-    InternalCache cache = (InternalCache) CacheFactory.getAnyInstance();
+    InternalCache cache = ManagementAgent.getCache();
```

Comment:
CacheServerBridge already has a reference to the cache, so you could replace the singleton caller to:
```
InternalCache cache = cache.getCacheForProcessingClientRequests();
```